src/rules/no-duplicate-selectors/README.md

```diff
@@ -60,6 +60,13 @@ a
   .foo {}
 ```
 
+```css
+a b {}
+a {
```

Comment:
Do you think we should use _direct nesting_ in this example? i.e.

```css
a b {}
a {
  & b {}
}
```

I believe we've used _direct nesting_ else where as it's common to `scss`, `less`, `postcss-nested` and `postcss-nesting`, and also it makes the example a bit more explicit.

With that in mind, is this section of the README:

> **If you are using a processor that modifies selectors, read this:** This rule will only compare the selectors that it sees; so as far as it's concerned, in `a { b {} & b {} }` there are no duplicate selectors. If you're using SCSS, though, you *do* have duplicates in the output: `a b {} a b {}`. If you want this rule to analyze the selectors in *your output*, make sure it runs *after* you have compiled your stylesheets.

still accurate now that `postcss-resolve-nested-selector` is being used?